touch_bar/pubspec.yaml

```diff
@@ -13,8 +14,16 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  touch_bar_platform_interface: ^0.2.0-alpha.1
-  touch_bar_macos: ^0.2.0-alpha.1
+  touch_bar_platform_interface:
+    git: 
+      url: https://github.com/Lynoma/touch_bar_flutter_plugin.git
+      ref: master
+      path: touch_bar_platform_interface/
+  touch_bar_macos: 
+    git: 
+      url: https://github.com/Lynoma/touch_bar_flutter_plugin.git
+      ref: master
+      path: touch_bar_macos/
```

Comment:
I configured the project to use [melos](melos.invertase.dev/) so now this will not be necessary anymore. Follow the instructions here to work with melos: https://github.com/vital-edu/touch_bar_flutter_plugin/blob/master/CONTRIBUTING.md#3-environment-setup